### PR TITLE
Rename: url -> configURL

### PR DESCRIPTION
**Pull request by npm1**
***
<a href="https://pr-preview.s3.amazonaws.com/npm1/FedCM/pull/284.html" title="Last updated on Jun 21, 2022, 8:20 PM UTC (ff642db)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/fedidcg/FedCM/284/41bd869...npm1:ff642db.html" title="Last updated on Jun 21, 2022, 8:20 PM UTC (ff642db)">Diff</a>

### CODE REVIEW

File: spec/index.bs [hunk 1]
```diff
@@ -320,7 +320,7 @@ could be implemented.
       mediated: “optional”, // “optional” is the default
       identity: {
         providers: [{
-          url: "https://idp.example/manifest.json",
+          configURL: "https://idp.example/manifest.json",
```
Comment (samuelgoto) [inline spec/index.bs:323]:
```suggestion
          config: "https://idp.example/config.json",
```

configURL is a bit long?

File: spec/index.bs [hunk 2]
```diff
@@ -356,7 +356,7 @@ could be implemented.
       mediation: "optional",
       identity: {
         providers: [{
-          url: "https://idp.example/manifest.json",
+          configURL: "https://idp.example/manifest.json",
```
Comment (samuelgoto) [inline spec/index.bs:359]:
```suggestion
          config: "https://idp.example/config.json",
```

File: spec/index.bs [hunk 3]
```diff
@@ -372,7 +372,7 @@ could be implemented.
       mediation: "required",
       identity: {
         providers: [{
-          url: "https://idp.example/manifest.json",
+          configURL: "https://idp.example/manifest.json",
```
Comment (samuelgoto) [inline spec/index.bs:375]:
```suggestion
          config: "https://idp.example/config.json",
```

File: spec/index.bs [hunk 4]
```diff
@@ -1087,7 +1087,7 @@ the [=Relying Party=] supports (e.g. that it has a pre-registration agreement wi
 
 <xmp class=idl>
 dictionary IdentityProvider {
-  required USVString url;
+  required USVString configURL;
```
Comment (samuelgoto) [inline spec/index.bs:1090]:
```suggestion
  required USVString config;
```

File: spec/index.bs (hunk 1 above)
Comment (npm1) [reply spec/index.bs:323]: 9 characters is too long? How so?

Comment (npm1) [reply spec/index.bs:323]: Discussed offline, keeping configURL

File: spec/index.bs (hunk 2 above)
Comment (npm1) [reply spec/index.bs:359]: There is a separate PR which updates the manifest file name (PTAL? :))

File: spec/index.bs (hunk 3 above)
Comment (npm1) [reply spec/index.bs:375]: See the other PR for the updated file name

File: spec/index.bs (hunk 2 above)
Comment (samuelgoto) [reply spec/index.bs:359]: SGTM

**Pull request merged**